x-pack/plugins/observability_solution/dataset_quality/public/hooks/use_overview_summary_panel.ts

```diff
@@ -26,10 +26,9 @@ export const useOverviewSummaryPanel = () => {
     .map((key: string) => services[key].length)
     .reduce((a, b) => a + b, 0);
 
-  const totalDocsCount = formatNumber(dataStreamDetails?.docsCount ?? 0, NUMBER_FORMAT);
+  const totalDocsCount = formatNumber(dataStreamDetails.docsCount, NUMBER_FORMAT);
```

Comment:
According to [this line](https://github.com/elastic/kibana/blob/49661a35744adea396d6a80039d1a5f9976c957e/x-pack/plugins/observability_solution/dataset_quality/server/routes/data_streams/get_data_stream_details/index.ts#L164) docsCount will always be a number, in the worst case scenario (no documents) it will be `0`.